packages/synthetic-chain/cli.ts

```diff
@@ -4,10 +4,13 @@ import { parseArgs } from 'node:util';
 import path from 'node:path';
 import { execSync } from 'node:child_process';
 import { buildProposalSubmissions, buildTestImages } from './src/cli/build.js';
-import { refreshDockerfile } from './src/cli/dockerfileGen.js';
+import { writeDockerfile } from './src/cli/dockerfileGen.js';
 import { matchOneProposal, readProposals } from './src/cli/proposals.js';
 import { debugTestImage, runTestImages } from './src/cli/run.js';
 
+// TODO change the tag to 'main' after multi-platform support https://github.com/Agoric/agoric-3-proposals/pull/32
+const baseImage = 'pr-32-linux_arm64_v8';
```

Comment:
Was this base image already in main!? How did an arch specific base image would work for non Mac users?

Reply:
I'm not sure. What's the concern?

Reply:
Can I run this on my x64 Linux machine ?

Reply:
Yes. With build/rebuild it won't matter because this value isn't used. With `append` it will work in CI and on x64 desktop because the base tag is x64. The only complication I can think of is for Mac users doing `append`, but I think Rosetta solves that.

Reply:
But that's the point I'm making, the base image / tag here is `arm64_v8` which wouldn't work on a x64 environment.

Reply:
Your OP is on an old commit. The tag in the PR has been `main` for 2.5 hrs https://github.com/Agoric/agoric-3-proposals/pull/52/commits/319ba1ff9647f5481c8e4655c4d8022e10f17557